src/Sulu/Bundle/WebsiteBundle/Twig/Seo/SeoTwigExtension.php

```diff
@@ -80,15 +80,15 @@ public function renderSeoTags(
     ) {
         $template = 'SuluWebsiteBundle:Extension:seo.html.twig';
 
-        @trigger_error(
+        @trigger_error(sprintf(
             'This twig extension is deprecated and should not be used anymore, include the "%s".',
             $template
-        );
+        ));
 
         $defaultLocale = null;
         $portal = $this->requestAnalyzer->getPortal();
         if ($portal) {
-            $defaultLocale = $portal->getXDefaultLocalization()->getLocale();
+            $defaultLocale = $portal->getDefaultLocalization()->getLocale();
```

Comment:
Why should we use the other default localization here? I thought the `x-default` was introduced especially for the SEO functionality.